lib/watir/locators/element/selector_builder/xpath.rb

```diff
@@ -74,7 +74,7 @@ def lhs_for(_building, key)
               # https://github.com/watir/watir/issues/72
               XpathSupport.downcase('@type')
             else
```

Comment:
What if we do:
```
when String
  "@#{key}"
```
`Symbol` always converts `_` to `-`, but `String` won't do any conversion